docs/wiki/99 - Developers/SQL Dictionary.md

```diff
@@ -47,3 +47,37 @@
 	| `OptimumMaxDays `    | Candidate's maximum age in days for visit to be flagged as _within **optimal** parameters of the study_   | The only effect of this field is a YES/NO flag showing up on the instrument_list page|
 	| `WindowMidpointDays `| Candidate's ideal age in days for a visit                                                                 |  |
 	
+
```

Comment:
@ridz1208 The table is nicely done. Did you use anything to generate it or did you type it all manually?

Reply:
manually